### ISSUE TITLE
Announce unread mention count

### ISSUE THREAD

**Issue by sukiletxe** [UnigramDev/Unigram#2325]
In Telegram for iOS, when navigating the chat list, if some unread chat has mentions too it is announced. It would be nice if this were implemented. (I recall this has been mentioned in insiders but cannot find an issue here)

**Comment by FrayxRulez** [comment]
This was done
